src/components/__tests__/TimeLineText.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TimeLineText from '../TimeLineText';
+import { mount } from 'enzyme';
+const DEFAULT_PROPS = {
+  title: 'OPORTUNIDADES ILIMITADAS',
+  text:
+    'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.' +
+    'Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.' +
+    'Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. ',
+  init: 'right',
+};
+
+describe('TimeLineText Component', () => {
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<TimeLineText {...DEFAULT_PROPS} />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+  it('should render children', () => {
+    const wrapper = mount(
+      <TimeLineText {...DEFAULT_PROPS}>
+        <div />
```

Comment:
Why should timeline text render a children if it seems it doesn't support printing children

Reply:
Fixed.